packages/unit-testing/stubs/log/logGlobalObject.js

```diff
@@ -0,0 +1,56 @@
+/**
```

Comment:
I don't know where this one comes from, I don't see it on the documentation. Also is not following AMD structure that starts with `define([...`

Reply:
This was exactly the omment I put in slack. 

`"Help mentions that log is both a global object but for testing purposes you could use N/log module. I have created the N/log module, but in the Suitescript 2.0 original library it appeared log as object. Do we use that instead?"`